spec/system/task_spec.rb

```diff
@@ -58,23 +56,22 @@
   end
 
   describe 'Task編集' do
+
+    let!(:other_task){create(:task, :task_date)}
```

Comment:
other_taskという変数名で定義しているので、同名の変数を呼び出さない限りは作成したテストデータを呼び出せていません。
`visit edit_project_task_path(project, task)`のコードでは最上部のletが呼ばれているはずなので、visit後にはTask.countの値が2になると思います。
必要ないテストデータは作成すると他の開発者が何か意味があるのかと混乱するので、let!を定義する位置を考え直した方が良いかと思います。

また行詰めやスペース区切りなどのフォーマット修正もお願いします。
```suggestion
    let!(:other_task){ create(:task, :task_date) }

Reply:
7b65eb3 でlet!の定義位置の修正を行いました。